tests/wpt/meta/css/css-cascade/layer-font-face-override.html.ini

```diff
@@ -1,9 +1,9 @@
 [layer-font-face-override.html]
```

Comment:
Please also open a bug describing this problem so that we can track fixing the promise returned by `FontFaceSet.load(...)`. In addition, there is syntax for linking to the bug in the ini file which we use in other places. I think it makes sense to do that here.